package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "npm run cypress:lint && npm run cypress:run",
     "build": "webpack --mode production",
-    "dev": "INSTRUMENTATION=1 webpack-dev-server --mode development",
```

Comment:
What is the way to provide environment variables in Windows?

Reply:
https://docs.oracle.com/en/database/oracle/r-enterprise/1.5.1/oread/creating-and-modifying-environment-variables-on-windows.html#GUID-DD6F9982-60D5-48F6-8270-A27EC53807D0 This link provides all the details you would need to provide environment variables inWindows!

Reply:
We want to keep this variable in place for now. You are free to get rid of it in your fork or find a way to make it work on Windows, in which case we could accept your change.

Reply:
This variable will remain in place. I figured it out to make it work on windows